projects/observability/src/shared/components/cartesian/d3/legend/cartesian-legend.ts

```diff
@@ -222,4 +321,8 @@ export class CartesianLegend {
   private isThisLegendEntryActive(seriesEntry: Series<{}>): boolean {
     return this.activeSeries.includes(seriesEntry);
   }
+
+  private isThisLegendSeriesGroupActive(seriesGroup: Series<{}>[]): boolean {
```

Comment:
Why do we care if a whole group is active? Are we styling the title? If so, why? If it's not interactive, we shouldn't change its color and instead leave it as a plain label.

Reply:
We do care about this, We can active whole group in once by clicking on the title!! That is why we need this as we have style changes as well like border change as well!!